src/App.svelte

```diff
@@ -217,8 +217,13 @@
 
 <Router>
   <BannerMessage
-    title="This software is a Beta release."
-    message={`The website is under construction, use at your own discretion.`}
+    title={`This software is a Beta release.`}
+    message={`The website is under construction, use at your own discretion`}
```

Comment:
Can we disable the first banner? I don't think we need it anymore and should only have 1 of them.

The publisher reward message below should use the golden styling.

Reply:
> although we are still in the Beta phase.

I would still keep it for now, We can just change the message to something like this maybe: "This software is a Beta release. Use at your own discretion".

<img width="576" alt="Screenshot 2023-02-21 at 11 41 40" src="https://user-images.githubusercontent.com/13741335/220391554-78f4b950-90b5-48be-86e9-0e14c0954e0d.png">

Reply:
Line 1 => Data Farming [is in Beta](https://blog.oceanprotocol.com/ocean-data-farming-is-launching-f633532d831c). Use at your own discrtion.
Line 2 => Publish your datasets trough our [marketplace](https://market.oceanprotocol.com/publish/1) to get [2X Rewards](https://blog.oceanprotocol.com/data-farming-publisher-rewards-f2639525e508) for publishers.